geomstats/_backend/pytorch/__init__.py

```diff
@@ -604,6 +613,8 @@ def copy(x):
 
 
 def cumsum(x, axis=None):
+    if not torch.is_tensor(x):
+        x = array(x)
```

Comment:
Yes, I had to add a similar condition in less_equal for torch. Why using `x = array(x)` here and not `x = torch.tensor(x)`, which could be faster? Do we need the logic of the `array` function?

Reply:
It depends on what kind of input this should accept. Say you want to do something like `np.cumsum([np.ones(3), np.ones(3)])`, then you will need the `array` function. Otherwise, `x = torch.tensor(x)` would be fine.

Reply:
Got it, thanks for explaining.